dpnp/backend/include/dpnp_gen_2arg_3type_tbl.hpp

```diff
@@ -111,8 +111,8 @@
 
 MACRO_2ARG_3TYPES_OP(dpnp_add_c,
                      input1_elem + input2_elem,
-                     sycl::add_sat(x1, x2),
-                     MACRO_UNPACK_TYPES(int, long),
+                     x1 + x2,
+                     MACRO_UNPACK_TYPES(int, long, bool),
```

Comment:
On Windows `long` and `int` map to the same 32-bit integer type. Perhaps we need `int` and `long long`.

Reply:
Thanks! Changed to `std::int32_t` and `std::int64_t`.